.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,107 @@
+name: 🐞 Report a bug
+description: Create a report to help us fix something that is not working correctly
+title: "Bug: <Brief description of the bug>"
+labels: ["Status: Needs Review", "Type: Bug"]
+assignees:
+  - nil
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing your ideas to make our project better!
+
+        To help us understand and evaluate your suggestion in a reasonable amount of time, please provide detailed information below.
+
+        If you have any questions or are unsure about anything, don't be afraid to ask! The maintainers are here to help.
+
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: "Please confirm the following prequisites are met:"
+      options:
+        - label: I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
+          required: true
+        - label: I have searched the existing issues to ensure this bug hasn't been reported before (please have a look through our [open issues list](https://github.com/TheOdinProject/javascript-exercises/issues) to make sure)
+          required: true
+        - label: "The title of this issue follows the `Bug - location of bug: brief description of bug` format, e.g. `Bug - Exercises: File type incorrect for all test files`"
+          required: true
+        - label: (Optional) I am interested in working on this issue and would like to be assigned to it
+          required: false
+
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      description: Clearly describe the bug. Include any screenshots that may help show the bug in action.
+    validations:
+      required: true
+
+  - type: textarea
+    id: bug-reproducement
+    attributes:
+      label: How to reproduce the bug
+      description: "List what steps one might need to take in order to reproduce this bug"
+      value: |
+        1. Step 1
+        2. Step 2
+        3. Step 3
+
+        E.g.:
+        1. Log in
+        2. Visit a lesson page
+        3. Click the complete button
+        4. The complete button does not update
```

Comment:
We should make this a placeholder instead (same for the other ones in this file and the feature request one.

Reply:
That's what I tried first with the checkboxes list. The list in the placeholder renders in a single line, which looks odd. I'll check what we can do about that.

Reply:
https://github.com/TheOdinProject/theodinproject/blob/main/.github/ISSUE_TEMPLATE/feature_request.yaml?plain=1

The website repo feature request form just does what you did but with `placeholder` instead of `value`. It initially renders with new lines but when you focus the textarea, it goes onto one line instead. Might not be much we can do about that but I'm not really concerned about it appearing on one line with a focused textarea.

Reply:
A bit disappointing, but oh well...